lmfit/model.py

```diff
@@ -1805,7 +1855,7 @@ def plot_residuals(self, ax=None, datafmt='o', yerr=None, data_kws=None,
         parse_complex : str, optional
             How to reduce complex data for plotting.
             Options are one of `['real', 'imag', 'abs', 'angle']`, which
-            correspond to the numpy functions of the same name.
+            correspond to the numpy functions of the same name. Default is 'abs'.
```

Comment:
for consistency with (hopefully) all other docstring, I would prefer to have this added as I suggested initially - in parenthesis. So here and for the other occurences below, it would become: ```correspond to the numpy functions of the same name (default is 'abs').```